### PR TITLE
Update overview.rst

### PR DESCRIPTION
**Pull request by ldehaan9**

### CODE REVIEW

**Review by mauriziobinello — CHANGES_REQUESTED** [review]
please open your pr against the development branch, and not master.

**Comment by ldehaan9** [comment]
Hi Maurizio,
If you think the changes I suggested are worthwhile, and you'd like me to review other documents, please feel free to send me the URLs and I'll take a look. Have a good day; be well.
 Laurens 

    On Tuesday, September 1, 2020, 01:01:14 AM PDT, Maurizio Binello <notifications@github.com> wrote:  
 
 


@mauriziobinello requested changes on this pull request.

please open your pr against the development branch, and not master.

File: docs/introduction/overview.rst [hunk 1]
```diff
@@ -2,27 +2,29 @@
 Before you start
 ################
 
-| This tutorial offers Java developers all the needed information to build a complete blockchain application on the Horizen Sidechain system.
+| This tutorial offers Java developers all the information needed to build a complete blockchain application on the Horizen Sidechain system.
```
Comment (Horusfeathers) [inline docs/introduction/overview.rst:5]: @zennermac LGTM

File: docs/introduction/overview.rst [hunk 2]
```diff
@@ -2,27 +2,29 @@
...
 
 | Apart from Java competency, this tutorial assumes that the reader has a high-level understanding of how blockchain-based distributed software works.
 
-So, concepts such as Transactions, UTXO's, Blocks, Validation, Confirmation, Consensus, Unique chain, and chain forks, Hash Function, Private/Public key, and Signing should be known and understood, as well as the concept of a network of nodes and node communication.
+So, you should be comfortable with concepts such as Transactions, UTXO's, Blocks, Validation, Confirmation, Consensus, Unique Chain, Chain Forks, Hash Function, Private/Public key, and Signing, along with the concept of a network of nodes and node communication.
```
Comment (Horusfeathers) [inline docs/introduction/overview.rst:9]: @zennermac 

You should be comfortable with concepts like transactions, UTXO's, blocks, validation, confirmation, consensus, unique chains, chain forks, hash functions, private/public key,s and signing, along with the concept of a network of nodes and node communication.

File: docs/introduction/overview.rst [hunk 3]
```diff
@@ -2,27 +2,29 @@
...
 
-If the above words are new to you, you can start by exploring the Horizen Academy website's material (`link <https://academy.horizen.global/>`_). Also, the original whitepaper by Satoshi Nakamoto, "Bitcoin: A Peer-to-Peer Electronic Cash System" (`link <https://bitcoin.org/bitcoin.pdf>`_), can be a good starting point. Direct experience with an existing blockchain software is also a very useful prerequisite. For that, you can install the Horizen "zend" software from (`Github <https://github.com/HorizenOfficial/zen>`_), and explore its rpc command interface and "regtest" mode.
+If the above words are new to you, you can start by exploring the Horizen Academy website's material (`link <https://academy.horizen.global/>`_). Also, the original whitepaper by Satoshi Nakamoto, "Bitcoin: A Peer-to-Peer Electronic Cash System" (`link <https://bitcoin.org/bitcoin.pdf>`_), can be a good starting point. Direct experience with an existing blockchain software is also useful. For that, you can install the Horizen "zend" software from (`Github <https://github.com/HorizenOfficial/zen>`_), and explore its RPC command interface and "regtest" mode.
```
Comment (Horusfeathers) [inline docs/introduction/overview.rst:11]: If the above terms are new to you, you can start by exploring the Horizen Academy website's material (`link <https://academy.horizen.global/>`_). Also, the original whitepaper by Satoshi Nakamoto, "Bitcoin: A Peer-to-Peer Electronic Cash System" (`link <https://bitcoin.org/bitcoin.pdf>`_), can be a good starting point. Direct experience with an existing blockchain software is also useful. For that, you can install the Horizen zend software from (`Github <https://github.com/HorizenOfficial/zen>`_), and explore its RPC command interface and "regtest" mode.

**Review by Horusfeathers — COMMENTED** [review]
@zennermac review completed.

**Review by zennermac — CHANGES_REQUESTED** [review]
> please open your pr against the development branch, and not master.

Please see the comments left by @Horusfeathers and make the requested changes.

**Pull request merged**